Documentation/ContentObjects/Extbaseplugin/Index.rst

```diff
@@ -62,6 +62,43 @@ Example
     page.10.extensionName = MyExtension
     page.10.pluginName = MyPlugin
 
+..  _cobj-extbaseplugin-examples-fluid:
+
+Example: Display an Extbase plugin in a Fluid template
+======================================================
+
+It is possible to display an Extbase plugin in Fluid using the
+:ref:`CObject ViewHelper <f:cObject> <t3viewhelper:typo3-fluid-cobject>`:
+
+..  literalinclude:: _CodeSnippets/_SomeTemplate.html
+    :caption: EXT:my_extension/Configuration/TypoScript/setup.typoscript
```

Comment:
The literal is a fluid template, but the caption is a path to a TypoScript file.

Reply:
Good catch! Replaced it